lib/Runtime/Debug/DebugDocument.cpp

```diff
@@ -158,6 +157,44 @@ namespace Js
         }
     }
 
+    Js::BreakpointProbe* DebugDocument::FindBreakpointId(StatementLocation statement)
```

Comment:
changed

---
In reply to: [62773771](https://github.com/Microsoft/ChakraCore/pull/926#discussion_r62773771) [](ancestors = 62773771)